jobsets/default.nix

```diff
@@ -57,6 +134,82 @@ let
     schedulingshares = 420;
   };
 
+  # Removes PRs which have any of the labels in ./pr-excluded-labels.nix
+  exclusionFilter = let
+    excludedLabels = import ./pr-excluded-labels.nix;
+    justExcluded = filter (label: (elem label.name excludedLabels));
+    isEmpty = ls: length ls == 0;
+  in
+    filterAttrs (_: prInfo: isEmpty (justExcluded (prInfo.labels or [])));
+
+  loadPrsJSON = path: exclusionFilter (builtins.fromJSON (builtins.readFile path));
+
+  # Make jobset for a project default build
+  mkJobset = { name, description, url, input, branch }: let
+    jobset = defaultSettings // {
+      nixexprpath = "release.nix";
+      nixexprinput = input;
+      inherit description;
+      inputs = {
+        "${input}" = mkFetchGithub "${url} ${branch}";
+      };
+    };
+  in
+    nameValuePair name jobset;
+
+  # Make jobsets for extra project branches (e.g. release branches)
+  mkJobsetBranches = { name, description, url, input }:
+    mapAttrsToList (suffix: branch:
+      mkJobset { name = "${name}-${suffix}"; inherit description url input branch; });
+
+  # Make a jobset for a GitHub PRs
+  mkJobsetPR = { name, description, input, modifier }: num: info: {
```

Comment:
`description` is not use. Should it be optional with default value `"PR ${num}: ${info.title}"`?